Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -63,21 +63,20 @@ public ApplicationContextManagerBlazor(IHttpContextAccessor hca, AuthenticationS
     /// </summary>
     protected HttpContext HttpContext { get; set; }
 
-    private void InitializeUser()
+    private async Task InitializeUser()
```

Comment:
Because of the change the ctor now produces 3 warnings. Please get rid of them with the discard syntax: `_ = InitializeUser()`.
I hate stuff like this in the ctor but I think we can't avoid it for now.

Reply:
Done...and done